structuralalignment/superposition/theseus.py

```diff
@@ -152,49 +105,66 @@ def _filemap(self):
         """
         Create filemap for Theseus
         """
-        filenames = self.fetched_pdbs_filename
         with open(self.filemap_file, "w") as outfile:
-            for fname in filenames:
+            for fname in self.pdbs_filename:
                 outfile.write(f"{fname} {fname}\n")
 
-    def _calculate(self, structures, **kwargs):
+    def _calculate(self, structures, identical: bool = True, **kwargs):
```

Comment:
Ideally we have two different _sub_-`_calculate_*` methods, depending on the value of `identical` we call one or another. Something like this:

```python
def _calculate(self, structures, identical: bool = True, **kwargs):
    # maybe here you can do some common ops for both strategies
    if identical:
        return self._calculate_identical(structures, **kwargs)
    else:
        return self._calculate_similar(structures, **kwargs)

def _calculate_identical(self, structures, **kwargs):
    # fill in

def _calculate_similar(self, structures, **kwargs):
    # fill in
```